Katello/katello
# Fixes #34962 - Details tab - Add Tracer to System properties card

#### What are the changes introduced in this pull request?
 Add Tracer to 'System properties' card in Details tab.

#### Considerations taken when implementing this change?
Related to https://github.com/theforeman/foreman/pull/9252

#### What are the testing steps for this pull request?

- Set "Show experimental labs" setting to Yes
- Check out the Foreman PR
- Check out the Katello PR
- Ensure that Tracer shows on the System Properties card in Details tab
